scss/bitstyles/utilities/block/block.stories.mdx

```diff
@@ -0,0 +1,40 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Utilities/Block" />
+
+# Block
+
+Force an element to `display: block`:
+
+- override a default-inline/flow element like a `<span>`
+- override `u-hidden` at a specific breakpoint
+
+
+<Canvas>
+  <Story name="Block">
+    {`
+      <span>Span 1 content</span>
+      <span class="u-block">Span 2 content, display: block</span>
+      <span>Span 3 content</span>
```

Comment:
Not sure if this is the best example here. Could make more sense to add 4th span, e.g. at position two, so that one actually sees a difference. With these three, it looks like all are displayed as block...

Reply:
Yes, fair point! Updated